biz.aQute.bndlib/src/aQute/bnd/help/instructions/ResolutionInstructions.java

```diff
@@ -34,18 +32,18 @@
 	 * Order the -runbundles sorted by name and merged with the existing value
 	 * if it exists. This is the default since it was the classic behavior.
 	 */
-	MERGESORTBYNAMEVERSION {
-		public boolean isMerge() {
-			return true;
-		}
-	};
-
-		public boolean isMerge() {
-			return false;
-		}
+	MERGESORTBYNAMEVERSION;
+	}
+
+	@interface RunStartLevel {
+		Runorder order() default Runorder.MERGESORTBYNAMEVERSION;
```

Comment:
Looks like you still have a `Runorder` type around even after renaming the instruction to runstartlevel. Perhaps Runorder is a subtype of Runstartlevel? Not sure of proper naming here.

Reply:
It is the enum for the run order.